src/System.Reflection.Emit.ILGeneration/tests/ILGenerator/ILGeneratorEmitWriteLine.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using TestLibrary;
+using System.Reflection;
+using System.Reflection.Emit;
+using Xunit;
+
+namespace System.Reflection.Emit.ILGeneration.Tests
+{
+    public class ILGeneratorEmitWriteLine
+    {
+        [Fact, ActiveIssue(1915, PlatformID.Linux)]
```

Comment:
PlatformID.AnyUnix rather than PlatformID.Linux, right? i.e. we expect this will fail on OSX right now, too.